Source/Core/VideoBackends/Vulkan/Renderer.cpp

```diff
@@ -767,9 +604,10 @@ void Renderer::DrawScreen(const TargetRectangle& scaled_efb_rect, u32 xfb_addr,
   vkCmdBeginRenderPass(g_command_buffer_mgr->GetCurrentCommandBuffer(), &info,
                        VK_SUBPASS_CONTENTS_INLINE);
 
-  // Draw guest buffers (EFB or XFB)
-  DrawFrame(m_swap_chain->GetRenderPass(), GetTargetRectangle(), scaled_efb_rect, xfb_addr,
-            xfb_sources, xfb_count, fb_width, fb_stride, fb_height);
+  // Draw
+  TargetRectangle source_rc = xfb_texture->GetConfig().GetRect();
+  BlitScreen(m_swap_chain->GetRenderPass(), GetTargetRectangle(), xfb_region,
```

Comment:
Noticed this after the fact when compiling on Linux, but was this supposed to be passed `source_rc` instead of `GetTargetRectangle()`'s result? `source_rc` seems to be unused.